samples-js/extensions.csproj

```diff
@@ -0,0 +1,19 @@
+
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  <WarningsAsErrors></WarningsAsErrors>
+  <DefaultItemExcludes>**</DefaultItemExcludes>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="host.json" />
+    <None Include="package.json" />
+    <None Include="local.settings.json" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="2.9.0" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="5.0.1" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Twilio" Version="3.0.2" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Script.ExtensionsMetadataGenerator" Version="4.0.1" />
+  </ItemGroup>
+</Project>
```

Comment:
Can we remove this an just rely on extension bundles? Keeping this would mean that users need to have the .NET dependency toolchain in their machine :)

Reply:
This is the only the new programming model can work right now because of #409. I want this directory to still be runnable until the fix is out, but I was planning on going back and reverting to extension bundles once #409 is fixed. I can file an issue for that too if it helps.

Also, what do you mean by users needing the dependency toolchain in their machine? Isn't this scenario covered by just `func extensions install`?

Reply:
Hmm, I'm not sure. I was under the impression that `func extensions install` might require some `dotnet` dependency on the given dev machine but now that I think about it, I have never seen it not work.

Let me put a pin on this, I realize bundles isn't viable just yet.